crazy_rl/multi_agent/numpy/catch/catch.py

```diff
@@ -17,7 +17,7 @@
 class Catch(BaseParallelEnv):
     """A Parallel Environment where drone learn how to surround a moving target trying to escape."""
 
-    metadata = {"render_modes": ["human", "real"], "is_parallelizable": True, "render_fps": 20}
+    metadata = {"render_modes": ["human", "real", "unity"], "is_parallelizable": True, "render_fps": 20}
```

Comment:
Shouldn't you this for the other envs too?

Reply:
If you delete the dlls on the executable side, the executable doesn't work. If the unity project dlls are removed, the user will have to install the pakages before being able to modify the unity project.

it's ok for the other envs

Reply:
@ElisaMrch seems ok for me to have to install dlls locally before being to launch as it is a repo containing the code, not off-the-shelve executables :). Plus it will greatly reduce the size of the PR :)

Reply:
so I delete executables and pakages? if I delete the executables, there will be a false line of code.

I'll update the documentation in any case